lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -977,7 +977,7 @@ def write_request_via_rest(entries:,
 
     def parse_json_or_nil(input)
       # Only here to please rubocop...
-      return nil if input.nil?
+      return input unless input.is_a?(String)
```

Comment:
Why not `return nil unless input.is_a?(String)`?

Reply:
I was debating between whether to collapse the Hash as we do if we managed to parse a hash from the JSON.

Sync'ed offline. We decided to leave the user input as it is.

Changed.

Reply:
Note that `input.is_a?(String)` being false did not necessarily mean that `input` is a Hash, so leaving it alone was the right choice.